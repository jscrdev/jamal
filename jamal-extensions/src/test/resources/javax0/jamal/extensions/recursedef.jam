{@define wilfred=define}{#{wilfred} alfred=wilfred}{alfred}
{@define black=white}{@define white=black}{{black}} {{{black}}}
{@define bla=whi}{@define ck=te}{{bla}{ck}} {{{bla}{ck}}}