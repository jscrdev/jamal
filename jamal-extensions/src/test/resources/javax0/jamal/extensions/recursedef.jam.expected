wilfred
black white
black white